changelog.txt

```diff
@@ -0,0 +1,4 @@
+changed - Emulators are not started unless specified.
```

Comment:
I have no idea what  this line means. Which emulators? All of them? Including functions? How does one specify?

Reply:
Specified which ones, and 'when' they're started. This isn't the place for documentation though - since I specify "serve" now, that should be enough to prompt anyone wondering to go "huh - `firebase serve --help`